src/audio_core/common.h

```diff
@@ -15,7 +15,7 @@ constexpr ResultCode ERR_INVALID_PARAMETERS{ErrorModule::Audio, 41};
 constexpr ResultCode ERR_SPLITTER_SORT_FAILED{ErrorModule::Audio, 43};
 } // namespace Audren
 
-constexpr u32_le CURRENT_PROCESS_REVISION = Common::MakeMagic('R', 'E', 'V', '9');
+constexpr u32_le CURRENT_PROCESS_REVISION = Common::MakeMagic('R', 'E', 'V', 'A');
```

Comment:
```suggestion
constexpr u32_le CURRENT_PROCESS_REVISION = Common::MakeMagic('R', 'E', 'V', ':');
```
Nintendo increments this byte with the ASCII value, so '9' + 1 = ':'

Reply:
Do you think I should change this to 0x3A instead of ':'? It's the reason I was confused, and I think having the byte written out would be more clear as to what's going here anyway.

Reply:
you would still need to cast in the case of the last parameter, since `u8 + int -> int`